### PR TITLE
Add Planet Page

### PR DESCRIPTION
**Pull request by MarwanYoussef22**

### CODE REVIEW

File: planet.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  const url = `${baseUrl}/planets/${planet?.id}/films`;
+  const films = await fetch(url)
+    .then(res => res.json())
+  return films;
+}
+
+const renderPlanet = planet => {
+  document.title = `SWAPI - ${planet?.name}`;  // Just to make the browser tab say planet name
+  nameH1.textContent = planet?.name;
+  const charactersLis = planet?.characters?.map(character => `<li><a href="/character.html?id=${character.id}"</li>`)
```
Comment (bentonsnodgrass) [inline planet.js]: const charactersLis = planet?.characters?.map(character => `<li><a href="/character.html?id=${character.id}">${character.name}</li>`)

Missing the end of the anchor tag and the string literal --  add: >${character.name}

File: planet.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  const sp = new URLSearchParams(window.location.search)
+  const id = sp.get('id')
+  getPlanet(id)
+});
+
+async function getPlanet(id) {
+  let planet;
+  try {
+    planet = await fetchPlanet(id)
+    planet.films = await fetchFilms(planet)
```
Comment (bentonsnodgrass) [inline planet.js:23]: After line 22, need to add planet.characters fetch

File: planet.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+let nameH1;
+let filmsDiv;
+let planetDiv;
+let charactersDiv;
```
Comment (bentonsnodgrass) [inline planet.js:4]: Change to filmsUL/characterUL

File: planet.js (hunk 1 above)
Comment (MarwanYoussef22) [reply planet.js]: Changes Done

File: planet.js (hunk 2 above)
Comment (MarwanYoussef22) [reply planet.js:23]: Changes Done

**Review by bentonsnodgrass — APPROVED** [review]

**Pull request merged**